docs/overrides.md

```diff
@@ -18,6 +18,9 @@ File and folder paths inside `.gitattributes` are calculated relative to the pos
 
 # Replace any whitespace in the language name with hyphens:
 *.glyphs linguist-language=OpenStep-Property-List
+
+# You can also use an alias of the language (eg. js instead of JavaScript)
+*.cshtml linguist-language=js
```

Comment:
```suggestion
# Language names are case-insensitive and may be specified using an alias:
*.cshtml linguist-language=js
```

Reply:
Good point! But I'd prefer to keep this info in a seperate line for more clarity. I've updated the code accordingly.

Reply:
This is an introductory section, so it behoves us to keep examples as concise as possible. If the wording I proposed earlier was unclear, perhaps try this instead:

~~~gitattributes
# Language names are case-insensitive and may be specified using an alias:
# the following three lines are all functionally equivalent:
*.es linguist-language=js
*.es linguist-language=JS
*.es linguist-language=JAVASCRIPT
~~~

Reply:
Ok, that multi-line example is clear enough. I've pushed this change with a slight punctuation adjustment to avoid using the colon twice.